multimap/src/main/java/org/infinispan/multimap/impl/function/PutFunction.java

```diff
@@ -0,0 +1,72 @@
+package org.infinispan.multimap.impl.function;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+import org.infinispan.commons.api.functional.EntryView;
+import org.infinispan.commons.marshall.AdvancedExternalizer;
+import org.infinispan.multimap.impl.ExternalizerIds;
+
+/**
+ * The adding function to remove a ket or a key/value on the multimap
```

Comment:
haha totally. Sorry guys I'm still so awful in english writing. Promise, I will improve !